test/test_dockerpy/README.md

```diff
@@ -2,29 +2,43 @@
 
 Python test suite to validate Podman endpoints using dockerpy library
 
+Validate :-
+
+Make sure podman binary exist under libpod/bin
+
+Validate Python 3.6+ installed using `python --version `
+
+Validate docker-py module exist by using `python list`. If missing install.
```

Comment:
```suggestion
Validate docker-py module exists by using `python list`. If missing install.
```

Reply:
I've never heard of `python list`, and it doesn't work for me:
```console
$ python list
python: can't open file 'list': [Errno 2] No such file or directory
```
Can you give more info about what `docker-py` is? I found and installed `python3-docker-4.2.0-1.fc32` but even with that, I get the following when I try running tests:
```console
$ python -m unittest discover test
...long python crap...
ImportError: cannot import name 'Client' from 'docker'
```
I think you have an environment that may be somewhat customized, and you're not aware of it.

Suggestion: I realize this is a lot of work, but one thing I do to help me place myself in the shoes of my target audience is to try my instructions on a fresh clean virt. That helps me catch assumptions I've made, so I can document them for others.